commons/src/main/java/fr/free/nrw/commons/category/CategorizationFragment.java

```diff
@@ -87,113 +95,72 @@ public void writeToParcel(Parcel parcel, int flags) {
         }
     }
 
-    private class CategoriesUpdater extends AsyncTask<Void, Void, ArrayList<String>> {
+    protected ArrayList<String> recentCatQuery() {
```

Comment:
Hm, I didn't write that either, just added the GPS cats to it. Do we want to put it in its own class anyway? Should it be a static class if we do?